packages/orbit-components/src/Separator/TYPESCRIPT_TEMPLATE.template

```diff
@@ -7,9 +7,19 @@ export type Align = "left" | "center" | "right";
 
 export type PaletteTokens =%TOKENS%
 
-export interface Props extends Common.SpaceAfter {
+export interface CommonProps extends Common.SpaceAfter {
   indent?: Indent;
-  type?: "solid" | "dashed" | "dotted" | "double" | "none";
-  color?: PaletteTokens;
   align?: Align;
 }
+
+type ConditionalSeparatorType =
+  | {
+      readonly color: PaletteTokens;
```

Comment:
I didn't know you'd accept the suggestion directly here 😅 so this one needs a change as well
```suggestion
      readonly color?: PaletteTokens;
```

Reply:
RIght :) 👍🏻